openassessment/assessment/test/test_peer.py

```diff
@@ -104,7 +104,7 @@
     }
 }
 
-# Answers are against RUBRIC_DICT -- this is worth 12 points
+# Answers are against RUBRIC_DICT -- this is worth 14 points
```

Comment:
I have no idea why this was incorrectly documented initially, but comparing `options_selected` to `RUBRIC_DICT` above yields 1+1+10+2=14 points, so I updated the documentation here to match.